textwithdrawable/src/main/java/wu/seal/textwithimagedrawable/BaseCombinedDrawable.java

```diff
@@ -0,0 +1,119 @@
+package wu.seal.textwithimagedrawable;
+
+import ohos.agp.components.element.Element;
+
+/**
+ * BaseCombinedDrawable.
+ */
+public class BaseCombinedDrawable extends Element {
+    private Element one;
+    private Element two;
+    private int relatedX;
+    private int relatedY;
+    private int paddingLeft;
+    private int paddingRight;
+    private int paddingTop;
+    private int paddingBottom;
+
+    public void createNativePtr(Object source) {
```

Comment:
What is the use of this method ? Check and remove if its not required for this library feature.